src/Collection/Collection.types.ts

```diff
@@ -9,16 +15,21 @@ export type CollectionAttributes = {
   minters: string[]
   managers: string[]
   forum_link?: string
-  lock?: Date
-  reviewed_at?: Date
+  lock: Date | null
+  reviewed_at: Date
```

Comment:
This is correct, it was a mistake on my part because node-pg used to return undefined for NULL values. But while we're at it, you mind fixing the type for reviewed_at and forum_link? both are NULLABLE and are not set at first

Reply:
Done!